data/en/cfpod.json

```diff
@@ -5,7 +5,7 @@
 	"returns": "",
 	"related": [],
 	"description": "Creates a pod, an area of the browser window or layout \n area with an optional title bar and body that contains \n display elements.",
-	"discouraged": "The use of tags generating UI is generally discouraged by the CFML community. See: http://static.raymondcamden.com/cfuitherightway/cfpod/index.html",
+	"discouraged": "The use of tags generating UI is generally discouraged by the CFML community. See: <a href=\"http://static.raymondcamden.com/cfuitherightway/cfpod/index.html\">http://static.raymondcamden.com/cfuitherightway/cfpod/index.html</a>",
```

Comment:
I would rather not have HTML in the document, as we do not know that HTML will always be used to render it.

Reply:
@pfreitag Okay, fixed it.

Reply:
Could we run `autoLink()` on it?